simulator_service/srv/SimulateTrajectory.srv

```diff
@@ -0,0 +1,4 @@
+trajectory_msgs/JointTrajectory trajectory
+bool wait_for_execution
```

Comment:
Suggestion: perhaps suffix `_completion` or a similar word to the name of this argument. This could be a non-native speaker thing, but to me 'waiting for execution' could also mean I want to wait until execution starts. From the code it looks like that is not the case.